insalan/user/tests.py

```diff
@@ -175,6 +174,39 @@ def send_valid_data(data, check_fields=[]):
             ],
         )
 
+    def test_register_bot_account(self):
+            """
+            Test registering valid users
+            """
+
+            def send_bot_data(data, check_fields=[]):
+                """
+                Helper function that will request a register and check its output
+                """
+                request = self.client.post("/v1/user/register/", data, format="json")
+
+                self.assertEqual(request.status_code, 400)
+                self.assertRaises(serializers.ValidationError)
+
+            send_bot_data(
+                {
+                    "username": "newplayer",
+                    "password": "1111qwer!",
+                    "password_validation": "1111qwer!",
+                    "email": "email@example.com",
+                    "name": "je suis un bot"
+                },
+                [
```

Comment:
We don't wanna check fields, you can remove that part (and the argument in `send_bot_data`)

Reply:
solved by [#2f4b2](https://github.com/InsaLan/backend-insalan.fr/pull/75/commits/2f4b2cfb49d91a7d7b973e3c55f704a5774d0a00)